src/main/java/com/minecolonies/coremod/colony/ColonyView.java

```diff
@@ -23,6 +24,7 @@
 import net.minecraft.client.Minecraft;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.client.Minecraft'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)